scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -48,6 +48,12 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
                                 </div>
                             </div>
                             <div class="field">
+                                <label translate>Desk language</label>
+                                <input type="text" class="line-input fullwidth-input"
```

Comment:
what about using languages from [settings](https://github.com/superdesk/superdesk-ntb/blob/master/server/settings.py#L194)?
I wouldn't get it from dictionaries, those should use same component to select language imo